mcs/class/Mono.Posix/Mono.Unix.Native/Syscall.cs

```diff
@@ -727,6 +727,194 @@ public enum MremapFlags : ulong {
 		MREMAP_MAYMOVE = 0x1,
 	}
 
+	[Map]
+	[CLSCompliant (false)]
+	public enum SockType : int {
```

Comment:
@knocte: Within reason...if the types aren't expected to be used together.

If the types *are* likely to be used together, then expecting everyone to add aliases is crazy. For example, adding a new `Array` type would cause instance collisions with `System.Array`, and thus would be a terrible name.

Now, if the namespaces were _never_ going to be used together, like e.g. WebForms + Mono.Posix or WPF + Mono.Posix, then collisions are entirely ignorable -- almost nobody will experience them.

In this case, we know that some projects *do* intermix `System.Net.Sockets` and `Mono.Unix`, so trying to avoid name collisions should be beneficial.